qutebrowser/completion/models/miscmodels.py

```diff
@@ -21,7 +21,10 @@
 
 from qutebrowser.config import configdata
 from qutebrowser.utils import objreg, log
+import itertools
```

Comment:
Please group the imports as per [pep8](https://www.python.org/dev/peps/pep-0008/#imports):

```
import itertools

from qutebrowser.… import …
```